lib/widgets/buttons/button.dart

```diff
@@ -53,10 +50,10 @@ class Button extends StatelessWidget {
                 onPressed: function,
                 child: Text(
                   label,
-                  textScaleFactor: 1.5,
                   style: TextStyle(
                     fontFamily: 'Raleway',
                     fontWeight: FontWeight.bold,
+                    fontSize: 22,
```

Comment:
ruh roh, the dynamic width formula depends on textScaleFactor, will formula still work ?

Reply:
test on both android/iOS

Reply:
i can change it back if you want

Reply:
just comment out textScaleFactor: 1.5, and leave fontSize: 22, and write a TODO next to it; if it becomes a problem, we'll know where to look